.limgo.json

```diff
@@ -0,0 +1,15 @@
+{
+  "coverage": {
+    "global": {
+      "statements": 0,
+      "lines": 0,
+      "branches": 0
+    }
+  },
+  "statistic": {
+    "excludes": [
+      "test_*",
+      "zz_generated.deepcopy.go"
+    ]
```

Comment:
Should we also exclude the `tests/` directory?

Reply:
oh, yeah, I didn't realize it had files with `test` as the prefix and some as the suffix.

Reply:
I added it.